openmls/src/schedule/mod.rs

```diff
@@ -293,13 +293,14 @@ impl InitSecret {
     /// group state.
     pub(crate) fn from_public_group_state(
```

Comment:
This should probably get a different name now. The comment needs updating as well.